github/Requester.py

```diff
@@ -174,6 +176,11 @@ def requestJsonAndCheck(self, verb, url, parameters=None, headers=None, input=No
     def requestMultipartAndCheck(self, verb, url, parameters=None, headers=None, input=None):
         return self.__check(*self.requestMultipart(verb, url, parameters, headers, input))
 
+    def requestBlobAndCheck(self, verb, url, parameters=None, headers=None, input=None):
+        o = urlparse.urlparse(url)
+        self.__hostname = o.hostname
+        return self.__check(*self.requestBlob(verb, url, parameters, headers, input))
```

Comment:
Shouldnt we change back the __hostname here? My subsequent api calls failed due to the host name assertion fails in requester..